### PR TITLE
jfsutils: new package at version 1.1.15

### PR DESCRIPTION
**Pull request by tslilc**
Please fill out the template.

Read: https://k1sslinux.org/wiki/kiss/style-guide

Please include the manifest for version bumps and new packages.

---

## Description of package

> jfsutils is a set of utilities for maintaining the JFS file system.

Analogous to e2fsprogs, but for JFS, required in particular for `mkfs.jfs` and `fsck.jfs`.

## Installed manifest of package

(`kiss manifest <pkg>`)

```
/var/db/kiss/installed/jfsutils/version
/var/db/kiss/installed/jfsutils/sources
/var/db/kiss/installed/jfsutils/patches/jfsutils-1.1.15-sysmacros.patch
/var/db/kiss/installed/jfsutils/patches/jfsutils-1.1.15-paths_h.patch
/var/db/kiss/installed/jfsutils/patches/jfsutils-1.1.15-linux-headers.patch
/var/db/kiss/installed/jfsutils/patches/jfsutils-1.1.15-gcc10.patch
/var/db/kiss/installed/jfsutils/patches/jfsutils-1.1.15-format-security-errors.patch
/var/db/kiss/installed/jfsutils/patches/jfsutils-1.1.15-check-for-ar.patch
/var/db/kiss/installed/jfsutils/patches/
/var/db/kiss/installed/jfsutils/manifest
/var/db/kiss/installed/jfsutils/checksums
/var/db/kiss/installed/jfsutils/build
/var/db/kiss/installed/jfsutils/README
/var/db/kiss/installed/jfsutils/
/var/db/kiss/installed/
/var/db/kiss/
/var/db/
/var/
/usr/share/man/man8/mkfs.jfs.8
/usr/share/man/man8/jfs_tune.8
/usr/share/man/man8/jfs_mkfs.8
/usr/share/man/man8/jfs_logdump.8
/usr/share/man/man8/jfs_fscklog.8
/usr/share/man/man8/jfs_fsck.8
/usr/share/man/man8/jfs_debugfs.8
/usr/share/man/man8/fsck.jfs.8
/usr/share/man/man8/
/usr/share/man/
/usr/share/
/usr/bin/mkfs.jfs
/usr/bin/jfs_tune
/usr/bin/jfs_mkfs
/usr/bin/jfs_logdump
/usr/bin/jfs_fscklog
/usr/bin/jfs_fsck
/usr/bin/jfs_debugfs
/usr/bin/fsck.jfs
/usr/bin/
/usr/
```

## New package

- [X] Latest upstream version.
- [X] I agree to maintain this package.
- [X] I agree to notify the @kiss-community maintainers if I can no longer maintain this package.

## Existing package

- [ ] I am the maintainer of this package.

### CODE REVIEW

**Comment by dilyn-corner** [comment]
Looks to me like this would require #758

**Comment by tslilc** [comment]
As ever you are correct and i was too excited and pushed this through quickly. Would it be satisfactory to have this depend on `util-linux` or should i check whether it's possible to avoid libuuid?

**Comment by dilyn-corner** [comment]
That PR *would* solve this indeed, and I had left it open hoping someone would submit something which needed it.

The exit message when the build fails suggests that `e2fsprogs` should be sufficient, but it apparently wasn't -- presumably `e2fsprogs-dev` would pull in `libuuid` in other distros, so the warning is just a least-effort catch all.

I leave it up to you whether or not it should require `util-linux` or `libuuid` -- I'm perfectly fine with just using `libuuid`. Minimal dependencies are a thing of beauty :) 

Next we just need someone to package `libblkid` and `zfs` can make an appearance in community...

**Comment by tslilc** [comment]
I'm happy to have this depend on `libuuid`, there's no need for the rest of `util-linux`. Unfortunately this drags in duplicate dependencies for some (including me), but i think the principle is clear. Hopefully we can refactor `util-linux` or i can take a closer look at those other things which seem to depend upon it.

File: community/jfsutils/build [hunk 1]
```diff
@@ -0,0 +1,15 @@
+#!/bin/sh -e
+
+for patch in *.patch; do
+	patch -p1 < "$patch"
```
Comment (dilyn-corner) [inline community/jfsutils/build:4]: Something I just noticed; this is a <TAB> not four spaces

File: community/jfsutils/depends [hunk 2]
```diff
@@ -0,0 +1 @@
+libuuid
```
Comment (dilyn-corner) [inline community/jfsutils/depends:1]: And I think this is probably just a make dep - can you test that to confirm?

**Review by dilyn-corner — CHANGES_REQUESTED** [review]

File: community/jfsutils/depends (hunk 2 above)
Comment (tslilc) [reply community/jfsutils/depends]: confirmed

File: community/jfsutils/build (hunk 1 above)
Comment (tslilc) [reply community/jfsutils/build]: wasn't aware of conventions, no problem

Comment (dilyn-corner) [reply community/jfsutils/build]: It's fine! The [style guide](https://github.com/kiss-community/wiki/blob/master/kiss/style-guide.txt) was removed from upstream and has yet to be replaced.

**Review by dilyn-corner — APPROVED** [review]

**Pull request merged**